src/components/Spinner.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import styled, { keyframes } from 'styled-components';
+import { IconLoaderOutline, BaseIconProps } from '@taskany/icons';
+
+const rotate = keyframes`
+    100% {
+        transform: rotate(360deg);
+    }
+`;
+
+const StyledSpinner = styled.div`
+    margin-top: -6px;
+    animation: ${rotate} 2s linear infinite;
+`;
+
+export const Spinner = ({ size = 'm', color, ...props }: BaseIconProps) => {
```

Comment:
May to add `duration` prop which allow customize animation duration, alos this prop must be optional with default value